py3dtilers/Common/tileset_creation.py

```diff
@@ -1,19 +1,17 @@
 import numpy as np
 from py3dtiles import B3dm, BatchTable, BoundingVolumeBox, GlTF, GlTFMaterial
```

Comment:
The three functions are bound together and only create_tileset() is called from other portions of the code.
Those functions could be static methods of a (static) class that is indeed a function aggregator whose purpose is to convert a GeometryTree instance to a TileSet instance. Following this logic, we could 
* call such a static class e.g. `FromGeometryTreeToTileset`
* rename create_tileset(...) to convert(...)
* make create_tile() and create_tile_content() private (with the conventional `__` prefix)

and then use that classe by writing something like
`tileset = FromGeometryTreeToTileset.convert(my_objects_to_tile_instance, that_extension)`